packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts

```diff
@@ -20,15 +19,15 @@ describe('Liveness Helper', () => {
     });
   });
 
-  describe('getFaceMatchStateInLivenessOval', () => {
```

Comment:
nit: if we plan to uncomment it in the future, shall we add a `TODO` to say when?